draft-ietf-quic-spin-exp.md

```diff
@@ -83,16 +107,16 @@ contacting the editor.
 # The Spin Bit Mechanism
 
 The latency spin bit enables latency monitoring from observation points on the
-network path. Since it is possible to measure handshake RTT without a spin
-bit, it is sufficient to include the spin bit in the short packet header. The
-spin bit therefore appears only after version negotiation and connection
-establishment are completed.
+network path throughout the duration of a connection. Since it is possible to
+measure handshake RTT without a spin bit, it is sufficient to include the spin
+bit in the short packet header. The spin bit therefore appears only after
+version negotiation and connection establishment are completed.
 
 ## Proposed Short Header Format Including Spin Bit
 
-As of the current editor's version of {{QUIC-TRANS}}, this proposal specifies
-using the fifth most significant bit (0x08) of the first octet in the short
-header for the spin bit.
+As of the current editor's version of {{QUIC-TRANSPORT}}, this proposal
+specifies using the fifth most significant bit (0x08) of the first octet in
```

Comment:
There's a mismatch with the figure (and the transport draft) here that specifies the sixth most significant bit (0x04) for the spin purpose.

Reply:
Oops. See #1295.